example/example.c

```diff
@@ -5,10 +5,14 @@
  * libsir.
  *
  * @author    Ryan M. Lederman \<lederman@gmail.com\>
- * @date      2018-2023
+ * @date      Copyright (c) 2018-2023
```

Comment:
I really wish I could jive with this, but I just can't get down wit' it. If there is documentation in Doxygen that says you should put a Copyright in @date, then I'll reconsider. What did it say before, just @date 2018-2013???

Reply:
SPDX and REUSE absolutely requires the Copyright (c) text somewhere.  If you don't want to put it there, then you'll have to duplicate it below that in the license area, like I had it before.

Or, you could do what all the other projects I've found that use Doxygen do that are SPDX/REUSE complaint (like FreeBSD, RTEMS, etc.) do, which is just not use those Doxygen fields at all, like ... 

![image](https://github.com/aremmell/libsir/assets/61629094/e78d58ab-527c-4fbc-ac72-b6b4317f4677)

The Doxygen documentation says that for the `@date`, `@copyright`, `@author` (and many others tags) that *"The text of the paragraph has no special internal structure"*, so there isn't any restriction on the text you can put there or how it's formatted.

Reply:
![image](https://github.com/aremmell/libsir/assets/61629094/9c93a2b6-1fde-4426-a1ef-87c171c13073)

![image](https://github.com/aremmell/libsir/assets/61629094/a26e6fed-ce19-465c-81ab-8f8f27e52de3)

Etc

Reply:
I would rather lose the Doxygen fields then and do it the way FreeBSD does.

Reply:
Ok I'll do that later

Reply:
Done